spec/motion-spec/matchers/respond_to_spec.rb

```diff
@@ -0,0 +1,48 @@
+# -*- encoding : utf-8 -*-
+describe 'Matcher::RespondTo' do
+  class TestCase
+    def call_me(a, b, c)
+    end
+  end
+  before { @test_object = TestCase.new }
+
+  context 'when subject responds to method name' do
+    it('passes') { expect(@test_object).to respond_to(:call_me) }
+
+    context 'with a given number of arguments' do
+      it 'passes when the right number' do
+        expect(@test_object).to respond_to(:call_me).with(3).arguments
+      end
+
+      it 'fails with the wrong number' do
+        message =
+          "#{@test_object.inspect} expected to respond to #call_me with 2" \
+          " arguments"
+
+        expect_failure(message) do
+          expect(@test_object).to respond_to(:call_me).with(2).arguments
+        end
+      end
+
+      it 'fails with the right number but asked not to' do
+        message =
+          "#{@test_object.inspect} not expected to respond to #call_me with 3" \
+          " arguments"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.